### PR TITLE
traverse upwards to find git repository root and add config repo tag, …

### PR DESCRIPTION
**Pull request by konz**
…see #56

### CODE REVIEW

**Comment by konz** [comment]
hm, pypy3 build fails on Travis at `pyb install_dependencies`. Pybuilder itself is not tested against pypy3. So it does not make much sense to support it, IMHO.

**Comment by esc** [comment]
Suggest not wasting too much time on this, feel free to just disable the pypy3 tests if it's not immediately obvious whats wrong.

**Comment by marco-hoyer** [comment]
i'm also ok with removing pypy3 as this doesn't seem to be a common platform

File: .travis.yml [hunk 1]
```diff
@@ -6,18 +6,17 @@ python:
   - '3.4'
   - '3.5'
   - "pypy"
-  - "pypy3"
 before_install:
   - git fetch --unshallow
 install:
   - pip install pip pybuilder coveralls --upgrade
-  - pyb install_dependencies
+  - pyb -X install_dependencies
```
Comment (marco-hoyer) [inline .travis.yml]: could you please remove the "-X" flag to minimize log jam?

**Pull request merged**